### PR TITLE
Port tgstation/tgstation#61081 - continuation of zewaka's 2021 port

### PR DESCRIPTION
**Pull request by MeggalBozale**
## About the PR 
~~Ports the tgui benchmark stuff in a weird hacky way~~

~~Uses IEChooser.exe (under MIT license, see https://github.com/LesFerch/LaunchIE/blob/main/LICENSE for the license and https://github.com/LesFerch/LaunchIE/tree/main for the project itself~~

~~IEChooser is used to deal with actually loading IE and not Edge, this could potentially be done w/o the executable but I chose this since it's the easiest and works.~~

Was done properly now

- [x] Automatically connect the browser and the benchmarking code
- [x] Close the IE window when complete

Continues #6272

## Why's this needed? 
With heavier and heavier UIs being TGUId, performance concerns are affecting gameplay -- we desperately need a tool to use for pinpointing hot code to optimize so players can continue to enjoy the TGUI windows.

### CODE REVIEW

**Comment by MeggalBozale** [comment]
I dont know if this is how I do it but i'm trying

**Comment by MeggalBozale** [comment]
lot of this will be spend finding a replacement for LaunchIE.exe

**Comment by MeggalBozale** [comment]
nvm that was easy

File: tgui/webpack.config.js [hunk 1]
```diff
@@ -125,13 +128,13 @@ module.exports = (env = {}, argv) => {
     ],
   };
 
-  // Add a bundle analyzer to the plugins array
-  if (argv.analyze) {
-    const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
-    config.plugins = [
-      ...config.plugins,
-      new BundleAnalyzerPlugin(),
-    ];
```
Comment (ZeWaka) [inline tgui/webpack.config.js:134]: ?

Comment (ZeWaka) [reply tgui/webpack.config.js:134]: i feel like there was some warning in the console about this

**Comment by MeggalBozale** [comment]
This needs a look at and is 'done'

**Pull request merged**